config/initializers/i18n_translation.rb

```diff
@@ -12,7 +12,15 @@ def t(key, options = {})
         i18_content = I18nContent.find_by(key: key)
         translation = I18nContentTranslation.find_by(i18n_content_id: i18_content&.id,
                                                      locale: current_locale)&.value
-        translation.presence || translate(key, options)
+        if translation.present?
+          if options.present?
```

Comment:
I'm not sure this condition is necessary :thinking:. What do you think about:

```ruby
        if translation.present?
          translation % options
        else
          translate(key, options)
        end
```

Reply:
I used the first option that seems simpler. The second one is also very hard to understand for me. 😂